GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
+
+    guard let dict = plist as? Dictionary<String, String> else {
+      throw StringsFileParserError.InvalidFormat
+    }
+
     var encoding: NSStringEncoding = NSUTF16StringEncoding
     let fileContent = try NSString(contentsOfFile: path, usedEncoding: &encoding)
+
+    func extractKey(line: String) -> String? {
```

Comment:
Right. The first look of this PR didn't have that `extractKey` and _line-by-line_ something. But after running test cases, I encountered that many cases had failed. That's because `NSDictionary` does not guarantee the order of keys. By order I mean the order that the key was added by.

I couldn't find a good way to preserve the original order of the keys. (We can't sort these keys in alphabetical order, right?😅)
So I decided reading the file line-by-line and extracting the keys, which seems quite redundant and ugly.

This code just works. Do you have any idea to make this good?